Makefile

```diff
@@ -0,0 +1,81 @@
+.PHONY: $(filter-out vendor bin/tools/psalm/vendor bin/tools/cs-fixer/vendor,$(MAKECMDGOALS))
+
+MYSQL_URL="mysql://root:root@mysql:3306/zenstruck_foundry?charset=utf8"
+POSTGRESQL_URL="postgresql://root:root@postgres:5432/zenstruck_foundry?charset=utf8"
+MONGO_URL="mongodb://mongo:mongo@mongo:27017/mongo?compressors=disabled&amp;gssapiServiceName=mongodb&authSource=mongo"
+SQLITE_URL="sqlite://./var/app.db"
+
+INTERACTIVE:=$(shell [ -t 0 ] && echo 1)
+ifdef INTERACTIVE
+	DC_EXEC=docker-compose exec
```

Comment:
yes for sure. `docker compose` is actually `docker-compose v2` I think it will gradually replace it. 
I guess I can make something like `docker --help | grep compose` to know which one to use.